### ISSUE TITLE
create project logo

### ISSUE THREAD

**Issue by tomkralidis** [geopython/pygeoapi#208]
We are in need of a project logo.  We are currently using the default Geopython logo as a placeholder (website, OSGeo page, etc.), but it would be valuable to have a dedicated project logo as part of marketing efforts.

**Comment by justb4** [comment]
Something like?

![pygeoapi-logo-robotoslab-world_200x200](https://user-images.githubusercontent.com/582630/63259616-335ed980-c27f-11e9-96bc-b0cb217bdefa.png)

(created in 2 min via https://freelogodesign.org selecting  "Technology", then a "World" icon (search) and `RobotoSlab` font.) for just €34,- we can have higher res for web etc.)

**Comment by pvgenuchten** [comment]
in dutch aapie (pronounced similar as API) means monkey, so a monkey in the logo would have my vote :-)

![image](https://cdn.pixabay.com/photo/2017/01/31/08/46/anthropomorphized-animals-2023331__340.png)

Alternative idea (similar to that of just): there seems some agreement to use a wheel logo for api's, in that scenario for example replace the inner circle with a globe. 

![image](https://user-images.githubusercontent.com/299829/63299789-ea3a7400-c2d6-11e9-851e-10f4a971d580.png)

or similar but with a pin (which is maybe better known as spatial data)

![image](https://user-images.githubusercontent.com/299829/63300266-14d8fc80-c2d8-11e9-95c7-dc3b3447e38e.png)

**Comment by tomkralidis** [comment]
+1 for the wheel logo with pin, both with and without the text.

**Comment by justb4** [comment]
Wheel is "cogwheel", actually was refering to Processing, but API: the better. Merging above ideas: resulted in this:

![pygeoapi-logo-robotoslab-cogwheel_gmap_200x200](https://user-images.githubusercontent.com/582630/63339835-0c1f0f80-c346-11e9-82e0-2fd946ca9b57.png)

(note to self: created on freelogodesign.org: 1) add icon search "cogwheel", then 2) closed shape Circle white, to cover inner cogwheel-circle, then 3) add pin icon, search for "google maps", group all, and 4) add Text "pygeoapi" use `RobotoSlab` font).

**Comment by tomkralidis** [comment]
+1. So can we do one icon with no text and another with text? Can we note the colour rgb value so we can align colours on website / docs?

We should also propose the logo on gitter and mailing list for decision I. The next day or two. 

Once decided, I can cover the high res cost. 

Target would be in time for our pygeoapi presentation at FOSS4G 2019 (next week).

**Comment by justb4** [comment]
* hex color is: **#1761a0**
* strictly for logos with/without text would be two logos to buy (2xEUR 34,-) but we get several formats, including SVG so just obtain the full logo and split icon off the text ourselves I expect.
* @tomkralidis can you put on Gitter/mailinglist?

**Comment by jorgejesus** [comment]
:+1: would prefer in black :)

**Comment by justb4** [comment]
@jorgejesus 
![image](https://user-images.githubusercontent.com/582630/63348621-e7cd2e00-c359-11e9-973e-dadfa815e867.png)

but maybe it is not clear what we are voting for, e.g. also the monkey? I can put them in order/numbered.

**Comment by francbartoli** [comment]
+1 for the black style

**Comment by justb4** [comment]
@francbartoli like this?
![image](https://user-images.githubusercontent.com/582630/63350753-498f9700-c35e-11e9-9656-31ff763e937e.png)

**Comment by francbartoli** [comment]
It's not bad but initially, I thought the pin form and the "py" in the name of different colors. Not sure now if it makes sense

**Comment by justb4** [comment]
Well, there's quite some permutations, like:
![image](https://user-images.githubusercontent.com/582630/63354720-48ae3380-c365-11e9-817b-2bc6619bd210.png)

and
![image](https://user-images.githubusercontent.com/582630/63354747-5368c880-c365-11e9-8f36-e12afd79b829.png)

**Comment by kalxas** [comment]
+1

**Comment by justb4** [comment]
yes, but +1 for which?

**Comment by francbartoli** [comment]
> yes, but +1 for which?

after all my +1 is for the first permutation with pin black and "py" blue

**Comment by kalxas** [comment]
I added a +1 to the post I liked best, but I like all of them

**Comment by justb4** [comment]
ok, thanks folks, I was a bit pressed for a conclusion as the freelogodesign session would time-out at some point and it takes quite some time to get the exact icons back (no exact science there). For blue: I propose to take the exact Python logo color:  Lapis Lazuli (hex: 306998) see [here](https://www.schemecolor.com/python-logo-colors.php). And the black a bit more mellow/grayish: hex: 3a3a3a. All in all this:

![image](https://user-images.githubusercontent.com/582630/63387299-5d122080-c3a5-11e9-9190-9d81d4803cad.png)

We can always change colors once the download is ready (and paid).

**Comment by justb4** [comment]
Ok this is the final version, had to take some shortcuts with the cogwheel on freelogodesign.org (transparancy stuff etc)

![medium](https://user-images.githubusercontent.com/582630/63393105-de73ae00-c3b9-11e9-8c63-d15196e4db19.png)

**Comment by justb4** [comment]
FYI the pygeoapi logo and icon files can be found at: https://github.com/geopython/pygeoapi/tree/gh-pages/static both transparent and non-transparent.

**Comment by tomkralidis** [comment]
Thanks @justb4.  Questions:

- are there SVG versions available?
- is there a favicon option/capability for the website?
- can we add a logo and favicon also to master `docs` for https://docs.pygeoapi.io ?

**Comment by justb4** [comment]
@tomkralidis yes, we have a complete design file package with several formats/resolutions, including SVG, about 600k. Thought I sent you that by e/gmail. I was still considering where to check this in in GitHub, maybe under GH pages branch /assets/logo?

No favicons yet, but these are easy to generate online. The logo includes icon+text but the icon is easily split-off for icon/favicon use.
